src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/controllers/CcdCallbackController.java

```diff
@@ -15,6 +16,14 @@
 
 @RestController
 public class CcdCallbackController {
+
```

Comment:
I think this controller needs refactoring you could check sscs bulk scan callback handling. you don't need to handle the callback type it will be always called as part of about to submit.

Reply:
sure its just a parameter on the url passed in right. Might as well check ?
If we get it in another state that would break our assumptions on correctness of the operation?